Kconfig

```diff
@@ -232,4 +232,12 @@ config PERFORMANCE_COUNTERS
 	  use the stamp() macro periodically to find out how long the cpu
 	  was in active/sleep state between the calls and estimate the cpu load.
 
+config POWER_STATE_COUNTERS
+	bool "Power state counters"
```

Comment:
Pls rename to "DSP residency counters". "Power state" suggests more complex state that includes other pieces of the DSP subsystem.